### PR TITLE
Working

### PR DESCRIPTION
**Pull request by jase-d**
Initialization of proxy repo
@bland0 
@javsflores

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "author": "Jase DiNardo",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Air-BnB-Mock-Up/jase-mosaic-proxy/issues"
+  },
+  "homepage": "https://github.com/Air-BnB-Mock-Up/jase-mosaic-proxy#readme",
+  "dependencies": {
+    "axios": "^0.20.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.4",
```
Comment (bland0) [inline package.json:22]: Should nodemon be a production dependency?

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+///////////////////////////////////////////
```
Comment (bland0) [inline server/index.js:1]: I'm not sure why index.js and httpHandler.js are separate files.

**Review by bland0 — CHANGES_REQUESTED** [review]
I think you're making it too complex. Consider that the role of the proxy server is to serve a single (template) file. What do you really need for that?

File: server/index.js (hunk 2 above)
Comment (jase-d) [reply server/index.js:1]: Good practice of separating modules

**Comment by jase-d** [comment]
> I think you're making it too complex. Consider that the role of the proxy server is to serve a single (template) file. What do you really need for that?

In my opinion that is all that is doing, 3 files that take the role of connection, httphandling, and the actual template. Can you elaborate more?

**Comment by bland0** [comment]
You have quite a few dependencies in your package.json I'm not sure you need, which bring in a large number of their own dependencies, and brings the size of your project up quite a bit.

**Comment by jase-d** [comment]
Oh very cool, yeah I took the unnecessary ones after I saw your comment, I
didn't have the correct plan in my head in the beginning so got a little
wild with them. I thought you were talking about the code itself.

Thanks for the advice!

**Pull request merged**